libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp

```diff
@@ -62,16 +62,24 @@ SortitionParamsManager::SortitionParamsManager(const addr_t& node_addr, Sortitio
   LOG_OBJECTS_CREATE("SORT_MGR");
   // load cache values from db
   params_changes_ = db_->getLastSortitionParams(config_.changes_count_for_average);
-  // restore VRF params from last change
-  if (!params_changes_.empty()) {
+  if (params_changes_.empty()) {
+    // if no changes in db save default vrf params
+    auto batch = db_->createWriteBatch();
+    SortitionParamsChange pc{0, sort_conf.targetEfficiency(), sort_conf.vrf};
```

Comment:
sort_conf is moved at this moment, use config_

Reply:
fixed